src/database/migrations/20240905093029-add_editing_key_in_resource_table.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		//to get to know the is user started editing when the resource status is in_review
+		await queryInterface.addColumn('resources', 'is_under_edit', {
+			type: Sequelize.BOOLEAN,
+			allowNull: true,
```

Comment:
@priyanka-TL Allow null need not be true right since the default value is false anyways